lib/galaxy/dependencies/conditional-requirements.txt

```diff
@@ -1,9 +1,9 @@
 # These dependencies are only required when certain config options are set
 psycopg2-binary==2.7.4
-WebError==0.10.3
-Pygments==2.2.0
+weberror==0.10.3
+pygments==2.3.1
```

Comment:
Does it still makes sense to have `pygments` here and `check_pygments()` in `lib/galaxy/dependencies/__init__.py` , since it is only a dependency of `weberror`?

There are other conditional dependencies (e.g. `docker`) requiring additional packages to be installed which are not in this file.

Reply:
Good idea, we can probably just drop it completely and it'll still be installed/used as necessary, I guess?

Reply:
Confirmed that this works on `master` and `release_XX.XX` branches (which do not install dev requirements):
```
Looking in indexes: https://wheels.galaxyproject.org/simple, https://pypi.python.org/simple
Collecting WebError==0.10.3 (from -r /dev/stdin (line 1))
  Downloading https://wheels.galaxyproject.org/packages/WebError-0.10.3-py2-none-any.whl (85kB)
    100% |████████████████████████████████| 92kB 1.1MB/s 
Requirement already satisfied: Tempita in ./.venv/lib/python2.7/site-packages (from WebError==0.10.3->-r /dev/stdin (line 1)) (0.5.2)
Collecting Pygments (from WebError==0.10.3->-r /dev/stdin (line 1))
  Using cached https://files.pythonhosted.org/packages/13/e5/6d710c9cf96c31ac82657bcfb441df328b22df8564d58d0c4cd62612674c/Pygments-2.3.1-py2.py3-none-any.whl
Requirement already satisfied: WebOb in ./.venv/lib/python2.7/site-packages (from WebError==0.10.3->-r /dev/stdin (line 1)) (1.8.2)
Requirement already satisfied: Paste>=1.7.1 in ./.venv/lib/python2.7/site-packages (from WebError==0.10.3->-r /dev/stdin (line 1)) (3.0.4)
Requirement already satisfied: six>=1.4.0 in ./.venv/lib/python2.7/site-packages (from Paste>=1.7.1->WebError==0.10.3->-r /dev/stdin (line 1)) (1.11.0)
Installing collected packages: Pygments, WebError
Successfully installed Pygments-2.3.1 WebError-0.10.3
```